dags/probe_scraper.py

```diff
@@ -22,7 +22,6 @@
     job_name="Probe Scraper",
     execution_timeout=timedelta(hours=4),
     instance_count=1,
-    owner="gfritzsche@mozilla.com",
```

Comment:
I'm confused why we're removing these owners

Reply:
They're already specified in the `default_args` above. We could remove the emails too by updating the defaults.